deisctl/cmd/cmd.go

```diff
@@ -268,18 +299,28 @@ func Install(targets []string, b backend.Backend, checkKeys func() error) error
 	return nil
 }
 
-func installDefaultServices(b backend.Backend, wg *sync.WaitGroup, outchan chan string, errchan chan error) {
+func installDefaultServices(b backend.Backend, stateless bool, wg *sync.WaitGroup, outchan chan string, errchan chan error) {
 
-	outchan <- fmt.Sprintf("Storage subsystem...")
-	b.Create([]string{"store-daemon", "store-monitor", "store-metadata", "store-volume", "store-gateway@1"}, wg, outchan, errchan)
-	wg.Wait()
+	if !stateless {
+		outchan <- fmt.Sprintf("Storage subsystem...")
+		b.Create([]string{"store-daemon", "store-monitor", "store-metadata", "store-volume", "store-gateway@1"}, wg, outchan, errchan)
+		wg.Wait()
+	}
 
 	outchan <- fmt.Sprintf("Logging subsystem...")
-	b.Create([]string{"logger", "logspout"}, wg, outchan, errchan)
+	if stateless {
+		b.Create([]string{"logspout"}, wg, outchan, errchan)
+	} else {
+		b.Create([]string{"logger", "logspout"}, wg, outchan, errchan)
```

Comment:
> Would that seem fair to undertake with this PR or would a followup be better (considering that we're close to PR freeze day)?

I thin we can probably make another pass at deisctl in general to clean up some repeated code. In this case, I preferred the verbosity of enumerating what exactly we're starting each time. It may be less code to do it your way, but I think it's more readable this way. Someone reading the code can see what exactly the `stateless` flag skips.